Makefile

```diff
@@ -38,9 +38,13 @@ lint: ## Run golangci-lin
 verify: tidy generate format lint ## Verify the current code generation and lint
 	git diff --exit-code
 
-build-cli:
+build-cli: ## Build the CLI binary
 	$(Q)go build -o combo
 
+TAG=quay.io/operator-framework/combo
```

Comment:
nit: This is more of an "image" than tag. I'd recommend either allowing users to override this default image, or allowing them to override the default repository + tag. This is personal preference so feel free to ignore: can we move this variable definition to the top of the Makefile?

Reply:
Updated!